src/api/CoreUtils.ts

```diff
@@ -82,23 +64,24 @@ export class CoreUtils {
                 const stdinReadError: IImperativeError = {
                     msg: "Error encountered while reading from stdin",
                     causeErrors: error,
-                    additionalDetails: (isNullOrUndefined(error)) ? undefined : error.message
+                    additionalDetails: error?.message
                 };
                 reject(stdinReadError);
             });
         });
     }
 
-    public static addLowerCaseKeysToObject(obj: any): any {
-        const result: any = {};
+    public static addLowerCaseKeysToObject(obj: IDatasetEntry | IUSSEntry): IDatasetEntry | IUSSEntry {
+
+        const result: { [key: string]: string } = {};
         for (const key of Object.keys(obj)) {
             // turn the object into a similar format to that returned by
             // z/osmf so that users who use the list ds command in main
             // zowe can use the same filtering options
             this.log.trace("Remapping key for data set to match core CLI. Old key '%s' New key '%s'", key, key.toLowerCase());
             result[key.toLowerCase()] = obj[key];
         }
-        return result;
+        return result as typeof obj;
     }
 
     public static async getProfileMeta(): Promise<ICommandProfileTypeConfiguration[]> {
```

Comment:
You might want to check if function getProfileMeta is used anywhere. If it is trying to read V1 profile meta file information, this function may no longer be necessary.

Reply:
Renamed to `getProfileSchema` and removed any references to V1 profiles 😋 
Good catch 😋